bodhi/server/models.py

```diff
@@ -2373,13 +2373,19 @@ def check_karma_thresholds(self, db, agent):
             self.comment(db, text, author=u'bodhi')
         elif self.stable_karma and self.karma >= self.stable_karma:
             if self.autokarma:
-                log.info("Automatically marking %s as stable" % self.title)
-                self.set_request(db, UpdateRequest.stable, agent)
-                self.request = UpdateRequest.stable
-                self.date_pushed = None
-                notifications.publish(
-                    topic='update.karma.threshold.reach',
-                    msg=dict(update=self, status='stable'))
+                if self.severity is UpdateSeverity.urgent or self.type is UpdateType.security:
+                    log.info("Automatically marking %s as stable" % self.title)
+                    self.set_request(db, UpdateRequest.stable, agent)
+                    # self.request = UpdateRequest.stable
+                    self.date_pushed = None
+                    notifications.publish(
+                        topic='update.karma.threshold.reach',
+                        msg=dict(update=self, status='stable'))
```

Comment:
Technically this event has also happened for the batched one in the ```else``` clause, so we can probably move this publish to be outside the ```if/else```.